_docs/style/colors.md

```diff
@@ -10,82 +10,553 @@ status: complete
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Health status iconography"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/color-palette/health-status-iconography.png" width="792"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Health status text
+
+You can pair a health status icon with text, but due to accessibility constraints, you cannot rely on text and text color alone to convey health status. You must always pair health status text with a health status icon.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Health status text"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/color-palette/health-status-text-usage.png" width="706"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Health status states
+
+In the default state, the critical, warning, and ok statuses map to the 500 band of red, yellow, and green, respectively.
+
+In the hover state, use the 700 band of red, yellow, and green.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Health status states"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/color-palette/health-status-states.png" width="484"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Non-health status
+
+Sometimes it is important to communicate the status of an entity, without also communicating urgency or criticality. The purpose of a non-health status indicator is to draw the attention of the user so that they can explore further, when desired.
+
+For example, use a badge or New! tag to indicate new items are available for review. These status labels use Orange 500.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Badge and New! tags"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/color-palette/non-health-status.png" width="1060"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Status labels
```

Comment:
we should probably cut this section and just have a status labels component page... thoughts?

do we need to say anything about them here? maybe we just note that for communicating other statuses, see xref-status Labels